lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -346,12 +346,21 @@
           <h2 class="heading -emphasized -inverse"><span><?php print t('Step 4: Prove It'); ?></span></h2>
         </div>
       </div>
+
       <div class="container__row">
           <div class="container__block">
             <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
             <?php if (isset($reportback_copy)): ?>
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
+
+            <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
+            <?php if (isset($helpful_tips)): ?>
+              <div class="with-lists">
```

Comment:
yup, I was just copying how we do other [markdown fields](https://github.com/DoSomething/phoenix/blob/dev/lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php#L151)   
![image](https://cloud.githubusercontent.com/assets/645205/18092319/dfdb9bb8-6e99-11e6-9cde-8edb7368681d.png)